angular-app/src/app/workspace/workspace.component.html

```diff
@@ -35,10 +35,9 @@ <h1 id="audio-name" class="center">Select an Audio File</h1>
             <div id="bottom-content">
                 <h1 id="lyric-doc-name" class="center">{{project.lyricsFileName}}</h1>
                 <br/>
-                <textarea id="lyric-doc" placeholder="Type Here..." rows="50" cols="100"></textarea>
+                <textarea id="lyric-doc" placeholder="Type Here..." rows="50" cols="100">{{project.lyricsContent}}</textarea>
```

Comment:
Note that this change isn't ideal either; if someone else edits the project in any way, the lyricsContent will be reset --- it shouldn't be an issue during demos, but I suspect having the lyrics as a separate document would probably be better in a production environment.

Reply:
@gwynethsc The lyrics will be reset because when other project details are being saved, you pull the current copy from the database?

If this is true, this seems like a flaw in the saving of a project that limits the intended used of Angular framework. There are several ways this could be remedied:
1.) Have a separate document for lyrics as you stated.
2.) providing UI popup that says all changes to the project will be saved and then saving every unsaved part of the project including lyricsContent changes.
3.) Do not pull the entire project again when you save. Just the parts you saved.

Reply:
I guess it would need some testing to be absolutely sure, but the reason I believe that changing part of something in Firestore updates the whole item is that when I had the projects overview in accordion view, adding a new project from one page snapped the accordion shut automatically in another page.